R/tm_g_ae_sub.R

```diff
@@ -344,13 +345,9 @@ srv_g_ae_sub <- function(id,
           bquote(group_labels <- setNames(.(group_labels), .(input$groups)))
         }
 
-        q1 <- teal.code::eval_code(
-          teal.code::new_qenv(tdata2env(data), code = get_code_tdata(data)),
-          code = group_labels_call
-        )
-        q2 <- teal.code::eval_code(q1, code = "")
+        teal.code::eval_code(data(), code = group_labels_call) %>%
```

Comment:
this would create styling issue GHC . could you run styler::style_file("R/tm_g_ae_sub.R") to fix this.

Reply:
Thanks for catching this. It would have also been missed by the CI because the `object_usage_linter` is set to NULL. I'll add this so we have this lint check done in the CI.